migtests/tests/pg/assessment-report-test-with-tdb/expectedAssessmentReport.html

```diff
@@ -837,188 +837,188 @@ <h2>Sharding Recommendations</h2>
                                 <td>
                                     <div class="scrollable-div">
                                         
-                                            public.test_exclude_basic<br>
```

Comment:
I think you have replaced the expected file directly, but ideally we should just make relevant changes as per the PR changes so it's easier to see what is changed in the PR

Reply:
Yeah agree, we should do that.

It was getting very tricky to update the report again and again.
I spent more than an hour on this but couldn't fix each line still. 
So went with this just replacing, considering the diff of reports i thoroughly reviewed at time of testing.

Each of these are around 15k lines long html
Maybe its worth considering the html reports for case where assessment doesn't have a lot of redundancies.

Thanks